src/pages/MyGardens/index.tsx

```diff
@@ -52,7 +72,22 @@ export const MyGardens = () => {
       exit={{ opacity: 0 }}
     >
       <div className="my-gardens-container">
-        <h1>My Gardens</h1>
+        <Grid
+          container
+          className={classes.myNiwaHeader}
+          direction="row"
+          justifyContent="space-between"
```

Comment:
Have you tried space-evenly? I think it might work better on a larger variety of screen ratios. I may be wrong though

Reply:
They both work in the exact same way but space-evenly puts equal space between and around items in a container where space-between puts the items at the start and end of the container. Screen ratios should not be a problem.